### PR TITLE
Fix pinch zoom on iOS

### PR DESCRIPTION
**Pull request by jakearchibald**
Well this was a pain in the arse to figure out. Thanks Safari.

### CODE REVIEW

File: src/client/lazy-app/Compress/Output/custom-els/PinchZoom/index.ts [hunk 1]
```diff
@@ -105,14 +106,23 @@ export default class PinchZoom extends HTMLElement {
...
         return true;
       },
       move: (previousPointers) => {
         this._onPointerMove(previousPointers, pointerTracker.currentPointers);
       },
+      // Unfortunately Safari on iOS has a bug there pointer event capturing
+      // doesn't work in some cases, and we hit those cases due to our event
+      // retargeting in pinch-zoom.
+      // https://bugs.webkit.org/show_bug.cgi?id=220196
+      avoidPointerEvents: isSafari,
```
Comment (jakearchibald) [inline src/client/lazy-app/Compress/Output/custom-els/PinchZoom/index.ts:125]: I added this option in the latest `pointer-tracker`.

File: src/client/lazy-app/Compress/Output/custom-els/PinchZoom/index.ts [hunk 2]
```diff
@@ -105,14 +106,23 @@ export default class PinchZoom extends HTMLElement {
...
+        ) {
           return false;
+        }
         event.preventDefault();
         return true;
       },
       move: (previousPointers) => {
         this._onPointerMove(previousPointers, pointerTracker.currentPointers);
       },
+      // Unfortunately Safari on iOS has a bug there pointer event capturing
```
Comment (AdriVanHoudt) [inline src/client/lazy-app/Compress/Output/custom-els/PinchZoom/index.ts]: s/there/where

Comment (jakearchibald) [reply src/client/lazy-app/Compress/Output/custom-els/PinchZoom/index.ts]:
```suggestion
      // Unfortunately Safari on iOS has a bug where pointer event capturing
```

**Review by surma — APPROVED** [review]
_Lovely_.

**Pull request merged**